capstone-rs/src/capstone.rs

```diff
@@ -342,6 +344,7 @@ impl Capstone {
         result
     }
 
+    #[cfg(feature = "not_diet")]
```

Comment:
I think we can avoid special casing this since the C library already checks if diet mode was enabled. I think we should avoid repeating logic in the C library where possible.

Reply:
My thoughts behind organising it like this is it incurs unecessary runtime cost (calling into the library, loading the diet mode global) for a value we can determine statically